orttraining/orttraining/test/gradient/gradient_checker.cc

```diff
@@ -125,7 +125,7 @@ inline Status GradientChecker<X_T, Y_T, JAC_T>::ComputeTheoreticalJacobianTransp
       continue;
     }
 
-    const size_t dy_size = y_infos[y_idx].shape.Size();
+    const int dy_size = static_cast<int>(y_infos[y_idx].shape.Size());
```

Comment:
keep dy_size as size_t, update c

Reply:
dy_size is used in one place but c is used in two places. I figured one static_cast is better than two. But can update c too.

---
In reply to: [417687388](https://github.com/microsoft/onnxruntime/pull/3743#discussion_r417687388) [](ancestors = 417687388)